sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsOptions.cs

```diff
@@ -43,6 +46,30 @@ public int MaxDegreeOfParallelism
             }
         }
 
+        /// <summary>
+        /// Gets or sets the number of times to try processing a message before moving it to the poison queue (where
+        /// possible).
+        /// </summary>
+        /// <remarks>
+        /// Some queues do not have corresponding poison queues, and this property does not apply to them. Specifically,
+        /// there are no corresponding poison queues for any queue whose name already ends in "-poison" or any queue
+        /// whose name is already too long to add a "-poison" suffix.
+        /// </remarks>
+        public int MaxDequeueCount
+        {
+            get { return _maxDequeueCount; }
+
+            set
+            {
+                if (value < 1)
```

Comment:
Are there any limits on the max value this can be?

Reply:
Not sure, in the QueuesOptions there wasn't a limit either.

Reply:
I don't believe there's a limit here, I guess it's Int64.MaxValue?